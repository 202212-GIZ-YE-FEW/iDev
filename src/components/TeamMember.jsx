import Image from "next/image";
const TeamMember = ({ name, job, photo }) => {
    return (
        <>
            {/* Single Card */}
            <div className='rounded-[25px] w-[11rem] bg-light-cyan'>
                <Image
                    src={photo}
                    alt={`${name} photo`}
                    width={176}
                    height={190}
                />
                <div className='w-full h-[5px] mt-0 bg-phosphorescent'></div>
                <div className='p-5'>
                    <h5 className='text-xl font-bold tracking-tight text-gray pb-[33px] h-[50px] mb-[40px] text-center'>
                        {name}
                    </h5>
                    <p className='font-normal text-gray pb-[40px] text-center'>
                        {job}.
                    </p>
                </div>
            </div>
        </>
    );
};

export default TeamMember;
