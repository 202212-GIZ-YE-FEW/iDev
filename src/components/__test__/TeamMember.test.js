import renderer from "react-test-renderer";
import TeamMember from "../TeamMember";

it("renders correctly", () => {
    const tree = renderer.create(<TeamMember />).toJSON();
    expect(tree).toMatchSnapshot();
});
