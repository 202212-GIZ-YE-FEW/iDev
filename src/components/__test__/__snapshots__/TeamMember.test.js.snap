// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rounded-[25px] w-[11rem] bg-light-cyan"
>
  <img
    alt="undefined photo"
    data-nimg="1"
    decoding="async"
    height={190}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src=""
    style={
      Object {
        "color": "transparent",
      }
    }
    width={176}
  />
  <div
    className="w-full h-[5px] mt-0 bg-phosphorescent"
  />
  <div
    className="p-5"
  >
    <h5
      className="text-xl font-bold tracking-tight text-gray pb-[33px] h-[50px] mb-[40px] text-center"
    />
    <p
      className="font-normal text-gray pb-[40px] text-center"
    >
      .
    </p>
  </div>
</div>
`;
